app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(part for part in split_integer(32, 6)) == 32
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 6
+    number_of_parts = 2
+    if value / number_of_parts == value // number_of_parts:
+        assert len(set(split_integer(value, number_of_parts))) == 1
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 8
+    assert split_integer(value, 1) == [value]
```

Comment:
```suggestion
    assert split_integer(8, 1) == [8]
```

no need to create variable if it use only once